src/QuantTypeWellFactories.js

```diff
@@ -91,4 +91,43 @@ class QPCR5ul {
   }
 }
 
-export { PlateReader, QPCR10ul, QPCR5ul }
+class TS10XVDJ {
+  constructor (cell = {}) {
+    this.filename = cell.filename
+    this.wellId = cell.wellId
+    this.sampleDescription = cell.sampleDescription
+    this.fromBasePair = cell.fromBasePair
+    this.toBasePair = cell.toBasePair
+    this.averageSizeBasePairs = cell.averageSizeBasePairs
+    this.concentration = cell.concentration
+    this.regionMolarity = cell.regionMolarity
+    this.percentageOfTotal = cell.percentageOfTotal
+    this.regionComment = cell.regionComment
+  }
+  get row () {
+    return this.wellId.match(/^[A-P]/).toString()
```

Comment:
Nor have I.  I came to the conclusion that it's only going to be up to P (row 16) since a 384 well plate has A to P and 1 to 24.  I think these plates are actually 96 well given the input data I was provided, but it could have been a limited example of the data.

Reply:
Often with QC steps a 96-well source plate is aliquoted into 3 quadrants of a 384-well QC plate (so 3 replicates from each well) and then additional controls are added into the 4th quadrant (a control concentration ladder to judge the concentration of the real samples against amongst other things). This replication more accurately allows you to determine the sample concentrations.
A 384-well plate is A1 to P24 as you say. 16 rows by 24 columns.
Not sure that is what's actually happening here but may explain it.